web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -398,7 +400,7 @@ const TeamReviews = ({ onBack, periodId }) => {
 
   const handleCloseDateChange = (val, period) => {
     const newDate = val?.$d;
-    const isoDate = newDate.toISOString() ?? null;
+    const isoDate = newDate?.toISOString() ?? null;
```

Comment:
If no launch date was specified, the `?` here prevents an error from trying to call `toISOString`.